### DISCUSSION TITLE
[Feature] Mass merge Face

### DISCUSSION THREAD

**Discussion by romain1206** [immich-app/immich#5269] [Feature Request]
### The feature

Is it possible to mass merge faces?
for example in this sreenshot it is the same person 4 times.

![image](https://github.com/immich-app/immich/assets/16817502/2039f42d-59d9-4ccf-9c7c-5f530705fd79)

I have other people also present several times

many thanks

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bmondream** [comment]
This seems like a duplicate of https://github.com/immich-app/immich/discussions/3894.

There's a feature for that already. Open a person's three dots menu and click `Merge person`

In case you need to merge more than five faces at once, I posted a guide in the mentioned discussion.
